libs/ui/src/components/shared/card.tsx

```diff
@@ -0,0 +1,77 @@
+import { Avatar, Center, Heading, Image, Stack, Text } from '@chakra-ui/react';
+import { FC } from 'react';
+import { Navigate } from '../Navigate';
+
+interface CardProps {
+  title: string;
+  description: string;
+  image: string;
+  link: string;
+  rounded: boolean;
+}
+
+export const Card: FC<CardProps> = ({
+  title,
+  description,
+  image,
+  link,
+  rounded,
+}) => {
+  return (
+    <Navigate href={link}>
+      <Stack
+        h="full"
+        bg="white"
+        shadow="lg"
+        rounded="lg"
+        overflow="hidden"
+        role="group"
+      >
+        <Center overflow="hidden">
+          {/* TODO Create shared image component */}
+          {rounded ? (
+            <Avatar
+              objectFit="cover"
+              boxSize={48}
+              src={process.env['NX_PUBLIC_URL'] + image}
+              transition="transform 0.5s ease-in-out"
+              _groupHover={{ transform: 'scale(1.1)' }}
+            />
+          ) : (
+            <Image
+              objectFit="cover"
+              h={48}
+              w="full"
+              src={process.env['NX_PUBLIC_URL'] + image}
```

Comment:
```suggestion
              src={image}
```

We could provide absolute image url when we use the component. Otherwise we will only be able to use local images.